internal/usecase/crypto.go

```diff
@@ -201,20 +203,21 @@ func (uc *CryptoUsecase) GetDataForEncryption(ctx context.Context, address strin
 	var identityPrefix shcrypto.Block
 
 	if len(identityPrefixStringified) > 0 {
-		identityPrefixBytes, err := hex.DecodeString(strings.TrimPrefix(identityPrefixStringified, "0x"))
-		if err != nil {
-			log.Err(err).Msg("err encountered while decoding identity prefix")
+		trimmedIdentityPrefix := strings.TrimPrefix(identityPrefixStringified, "0x")
+		if len(trimmedIdentityPrefix) != 2*IdentityPrefixLength {
+			log.Warn().Msg("identity prefix should be of length 32")
 			err := httpError.NewHttpError(
-				"error encountered while decoding identity prefix",
+				"identity prefix should be of length 32",
```

Comment:
Could you also update the comment here for clarity to: "identity prefix should be of byte length 32"?